Sources/Sentry/SentryReplayApi.m

```diff
@@ -22,18 +23,49 @@ - (void)unmaskView:(UIView *)view
 
 - (void)pause
 {
-    SentrySessionReplayIntegration *replayIntegration =
-        [SentrySDK.currentHub getInstalledIntegration:SentrySessionReplayIntegration.class];
+    SentrySessionReplayIntegration *replayIntegration
+        = (SentrySessionReplayIntegration *)[SentrySDK.currentHub
+            getInstalledIntegration:SentrySessionReplayIntegration.class];
     [replayIntegration pause];
 }
 
 - (void)resume
 {
-    SentrySessionReplayIntegration *replayIntegration =
-        [SentrySDK.currentHub getInstalledIntegration:SentrySessionReplayIntegration.class];
+    SentrySessionReplayIntegration *replayIntegration
+        = (SentrySessionReplayIntegration *)[SentrySDK.currentHub
+            getInstalledIntegration:SentrySessionReplayIntegration.class];
     [replayIntegration resume];
 }
 
+- (void)start
+{
+    SentrySessionReplayIntegration *replayIntegration
+        = (SentrySessionReplayIntegration *)[SentrySDK.currentHub
+            getInstalledIntegration:SentrySessionReplayIntegration.class];
+
+    if (replayIntegration == nil) {
```

Comment:
`h`: Is it really intended to install the integration the user didn't decide to enable it in the first place? IMO, this should be a NoOP, if SessionReplay isn't installed.

Reply:
yeah, i imagine the whole integration pattern as our way of letting users set things that will go on to work automatically after that. in this kind of API, start/stop should be alternative API to the integration, not ways of controlling an integration.

Reply:
The start function is a way of manually control SR even when the integration was not started, so we can't use a NoOP.
SentryReplayApi is just a dummy interface to bridge commands over the session replay, we need a place to hold reference of session replay where we can use if later.
A late installed integration is as good as any other approach, and already have all the code to control the replay.

Reply:
Okay again another pain point of our integrations. It's confusing, but out of scope of this PR.